neynar-webhook-kafka-consumer/.gitignore

```diff
@@ -0,0 +1,11 @@
+dist/
+node_modules/
+.env
```

Comment:
maybe do `/.env*` to cover all of these

Reply:
but it will also ignore `.env.example`

Reply:
I usually name that "env.example" so that it stands out clearly for users in their file browsers